win32ss/drivers/videoprt/registry.c

```diff
@@ -565,10 +565,10 @@ IntCreateNewRegistryPath(
                                 OBJ_KERNEL_HANDLE | OBJ_CASE_INSENSITIVE,
                                 NULL,
                                 NULL);
-    Status = ZwOpenKey(&NewKey, KEY_READ, &ObjectAttributes);
+    Status = ZwOpenKey(&NewKey, KEY_READ | KEY_WRITE, &ObjectAttributes);
     if (!NT_SUCCESS(Status))
     {
-        ERR_(VIDEOPRT, "Failed to open settings key. Status 0x%lx\n", Status);
+        ERR_(VIDEOPRT, "Failed to open NewKey. Status 0x%lx\n", Status);
```

Comment:
The change itself is fine with me, though I think these strings could stay as is (= identical).
Yet, if you change them, update the 3rd occurrence too.

Reply:
Pleases reference a Line Number where the 3rd occurrence happens. Thanks.

Reply:
Line 332.